frontend/src/static/js/v_summary.js

```diff
@@ -251,6 +251,19 @@ window.vSummary = {
       return res;
     },
 
+    getRepoLink(repo) {
+      const { REPOS } = window;
+      const urlString = repo.location;
+
+      if (urlString.includes('://github.com')) {
```

Comment:
following the previous conversation, let's change it to check if it has a valid organization.